### PR TITLE
ci: fix the cleanup step in the rc-release and publish-release pipelines

### PR DESCRIPTION
**Pull request by gzukel**
# Description

fix: fix the cleanup step in the rc-release and publish-release pipelines

Closes: No-Ticket-Adhoc-Fix

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions 

# Checklist:

- [x] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: .github/workflows/rc-release.yml [hunk 1]
```diff
@@ -85,4 +85,4 @@ jobs:
       - name: Clean Up Workspace
         if: always()
         shell: bash
-        run: rm -rf *
\ No newline at end of file
+        run: sudp rm -rf * || echo "failed to clean workspace please investigate"
```
Comment (lumtis) [inline .github/workflows/rc-release.yml:88]:
```suggestion
        run: sudo rm -rf * || echo "failed to clean workspace please investigate"
```

File: changelog.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Getting the correct TSS address for Bitcoin now requires proviidng the Bitcoin c
 * added metrics to track the burn rate of the hotkey in the telemetry server as well as prometheus
 
 ### Fixes
-
+* Fix release pipelines cleanup step.
```
Comment (lumtis) [inline changelog.md:35]: Maybe we can wait for `v12` to be merged and put it in `unreleased`

**Review by lumtis — COMMENTED** [review]
I think we can rename `fix` to `ci`

**Review by billyyang423 — APPROVED** [review]

**Review by CharlieMc0 — APPROVED** [review]
Approved from DevOps. Timing for merging and change log refer to @lumtis

File: changelog.md (hunk 2 above)
Comment (lumtis) [reply changelog.md:35]: Fine to keep it, we can manage conflicts afterward

**Review by lumtis — APPROVED** [review]

File: changelog.md (hunk 2 above)
Comment (lumtis) [inline changelog.md:35]: Can we add it above under `Unreleased` and precise the PR link in the entry?

**Pull request merged**